src/Common/src/Interop/Windows/mincore/Interop.GetLongPathNameW.cs

```diff
@@ -8,10 +9,30 @@ partial class Interop
 {
     partial class mincore
     {
+        /// <summary>
+        /// WARNING: This overload does not implicitly handle long paths.
+        /// </summary>
```

Comment:
Ditto on using a different name.  Do we expect any code other than the other overload to use these non-extending versions?  Let's make them private if possible.